### DISCUSSION TITLE
Infinite Loading For NM Optimization

### DISCUSSION THREAD

**Discussion by nbungart1999** [phoebe-project/phoebe2#466] [Questions]
When I try to run the nelder mead solver I get stuck in this warning loop saying my times are outside of interpolation limits.
![image](https://user-images.githubusercontent.com/83960306/121725118-d5842d00-cab6-11eb-8749-8cf047030b01.png)

**Comment by kecnry** [comment]
This warning is just saying that the computation of residuals is falling back on interpolating in phase-space instead of time-space (since you likely set `compute_phases`).  The warning itself is slated for removal in the 2.4 release since it does get overly verbose when fitting, but you can hide it by setting your logger at the error level instead if you'd rather.  

The first iteration will take a number of forward-model evaluations, so seeing even up to ten (or twenty since it looks like you have two datasets) of these before completing the first iteration isn't that concerning.  Does it eventually get past the first iteration?  If you have an expensive system, you may want to use [export_solver](http://phoebe-project.org/docs/2.3/tutorials/export_solver) and run that script within MPI and/or consider disabling unneeded effects (irradiation, distortion, etc).

It looks like the forward model itself is succeeding (so your parameters are passing checks and any priors you might have set), otherwise it would never get as far as this warning message.  If it still never progresses, you can send me the bundle file (just call b1.save right before this line) and I'll see if I can reproduce and track down what is happening.

**Reply by nbungart1999** [reply]
Ok, I just enabled ellc so I will see if that speeds it up. My advisor thinks that the optimization step just takes a while so I'm going to run it using compute='fastcompute' and see what happens from there. If nothing happens I'll send the notebook. This notebook is shorter because I exported the bundle to a new notebook so I wouldn't have to run a big notebook every time I change things.

**Reply by kecnry** [reply]
Ok - just make sure to occasionally compare back to phoebe and make sure the models agree well enough.  ellc (and the mapping of parameters from phoebe to ellc) make some approximations that are not always valid, especially for eccentric and highly distorted systems.

**Reply by nbungart1999** [reply]
Ok, I figured out what was happening. I actually have no error it is just super long to run. The maxiter was set to 10000 so it was just taking far to long to run and I got impatient and thought there was some error. After changing the maxiter to a lower value such as 100 it did end up finishing and I got some good looking plots
